b2share/modules/b2share_main/templates/b2share_main/page.html

```diff
@@ -0,0 +1,100 @@
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div id="page"></div>
+            </div>
+
+            <div id="footer">
+                <div class="footer">
+                    <div class="container-fluid">
+                        <div class="row">
+                            <div class="col-md-4 align-self-center">
+                                <div class="logo">
+                                    <a href="https://eudat.eu"><img src="/static/img/logo_eudat_cdi.svg" alt="EUDAT CDI logo" style="max-width: 200px"></a>
+                                    <p>© 2020 EUDAT Collaborative Data Infrastructure </p>
+                                </div>
+                            </div>
+                            <div class="col-md-8">
+                                <div id="menu">
+                                    <ul>
+                                        <li><a href="https://hdl.handle.net/11304/e43b2e3f-83c5-4e3f-b8b7-18d38d37a6cd">EUDAT Service ToU</a></li>
+                                        <li><a href="https://hdl.handle.net/11304/e43b2e3f-83c5-4e3f-b8b7-18d38d37a6cd">Terms of Use</a></li>
+                                        <li><a href="https://eudat.eu/privacy-policy-summary">Data Privacy Statement</a></li>
+                                        <li><a href="https://eudat.eu/what-eudat">About EUDAT</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="secondary-footer">
+                    <div class="container-fluid">
+                        <div class="row">
+                            <div class="col-md-8">
+                                <div class="images">
+                                    <img src="/static/img/eu.jpg" alt="EU">
+                                    <img src="/static/img/eosc-hub-g.png" alt="EOSC-HUB" style="max-width: 150px">
+                                </div>
+                                <p>
+                                    B2SHARE is co-funded by the EOSC-hub project (Horizon 2020) under Grant number 777536
+                                </p>
+                            </div>
+                            <div class="col-md-4 align-self-center">
+                                <div class="pull-right">
+                                    <p>B2SHARE v.<span id="version"></span><br/>Powered by &nbsp;<img src="/static/img/invenio.png"></p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script src="/static/b2share-bundle.js"></script>
+</body>
```

Comment:
Currently we are using a webpack configuration that adds the hash to the filename.